doc/source/getting_started/WorkingWithInferenceData.ipynb

```diff
@@ -13,11 +13,13 @@
   },
```

Comment:
<p>We could use a note directive here so it gets better formatting.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz/pull/1697/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>